.github/workflows/release-on-tag.yml

```diff
@@ -0,0 +1,25 @@
+name: Create new empty release
+
+on:
+  push:
+    tags:
+      - '*'
+
+jobs:
+  build:
+    name: Create Release
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@master
```

Comment:
What is the purpose of checking out the code?

Reply:
you are right, it is not needed, I will remove it